scss/_base_forms.scss

```diff
@@ -7,10 +7,8 @@
 /// Form element styles
 /// @group Forms
 @mixin maas-forms {
-
-  // All input elements
+  //scss-lint: disable no-vendor-prefixes
```

Comment:
This should be disabled around the thing its disabling so we dont leave it and get confused.